roles/cloud-bootstrap/tasks/network.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Create control-plane network
+  os_network:
+    cloud: "{{ cloud }}"
+    name: control-plane
```

Comment:
@gandelman-a cool. I'll switch up the order of `accounts.yml` and `network.yml` and add the auth parameters.